pkg/pillar/types/certinfotypes.go

```diff
@@ -5,16 +5,46 @@ package types
 
 import (
 	"encoding/hex"
+	"github.com/google/go-tpm/tpmutil"
+)
+
+// hashing algorithm used for a payload
+type ZHashAlgorithm uint8
+
+const (
+	HASH_ALGORITHM_NONE ZHashAlgorithm = iota + 0
+	HASH_ALGORITHM_SH256_16BYTES
+	HASH_ALGORITHM_SH256_32BYTES
+)
+
+// various certificate types
+type ZCertType uint8
+
+const (
+	CERT_TYPE_NONE ZCertType = iota + 0
+	CERT_TYPE_CONTROLLER_SIGNING
+	CERT_TYPE_CONTROLLER_INTERMEDIATE
+	CERT_TYPE_CONTROLLER_ECDH_EXCHANGE
```

Comment:
Don't put controller ones here. The existing type CertType seems to have the values needed for the device side. So a new type is not needed.